src/catkin_pkg/changelog_generator_vcs.py

```diff
@@ -128,7 +128,7 @@ def _truncate_timestamps(self, tags):
 
 class GitClient(VcsClientBase):
 
-    type = 'git'
+    client_type = 'git'
```

Comment:
I would prefer suppressing the warning rather then changing the API for style reasons.